rotary/config.yaml

```diff
@@ -0,0 +1,118 @@
+# Config file for rotary
+
+## Directory for database installation
+db_dir: "/Data/databases/rotary"
+
+## Resources
+threads: 20
+# Memory in Gigabytes
+#   At least ~55-60 GB is needed
+memory: 100
+
+## Long read QC
+# Minimum read length
+minlength: 1000
+# Minimum average quality (PHRED) score. Q13 is ~5% expected error rate.
+minavgquality: 13
+
+## Short read QC
+
+## Adapter trimming
```

Comment:
Should we consider adding a simple setting to completely skip short and/or long read QC?

Reply:
@jmtsuji I made this into an issue.